Dockerfile

```diff
@@ -0,0 +1,19 @@
+# A docker image to help with kerberos for testing
+FROM centos/nodejs-6-centos7
```

Comment:
Only works with nodejs-8-centos7 as parent image. No idea why is that so but it needs to be changed here.